apps/web/src/app/[locale]/challenge/_components/comments/comment.tsx

```diff
@@ -198,34 +212,35 @@ export function Comment({
         </div>
       ) : null}
 
-      {!isFetching && showReplies && data?.pages.at(-1)?.hasMore ? (
-        <Button
-          variant="ghost"
-          className="gap-1 text-xs text-neutral-500 duration-200 hover:text-neutral-400 dark:text-neutral-400 dark:hover:text-neutral-300"
-          onClick={() => fetchNextPage()}
-        >
-          <MoreHorizontal size={24} />
-          Load More
-          <span className="sr-only">Load More</span>
-        </Button>
-      ) : null}
-
-      {allReplies.isLoading && allReplies.fetchStatus !== 'idle' ? <CommentSkeleton /> : null}
+      {isLoadingReplies && fetchStatus !== 'idle' ? <CommentSkeleton /> : null}
       {showReplies ? (
-        <div className="flex flex-col-reverse gap-1 pl-6 pt-1">
-          {data?.pages.flatMap((page) =>
-            page.comments.map((reply) => (
-              // this is a reply
-              <SingleComment
-                comment={reply}
-                isReply
-                key={reply.id}
-                replyQueryKey={replyQueryKey}
-                preselectedCommentMetadata={preselectedCommentMetadata}
-              />
-            )),
-          )}
-        </div>
+        <>
+          <div className="flex flex-col gap-1 pl-6 pt-1">
```

Comment:
I swapped `flex-col-reverse` for `flex-col`. This puts newest replies on top, older replies going down. Pressing "Load More" gets the next page of older replies. This matches LeetCode

For example, https://leetcode.com/problems/two-sum/description/comments/1565865
<img width="350" alt="image" src="https://github.com/typehero/typehero/assets/7817695/0c94f4fc-b888-448b-9d68-44136a6f7c8e">

Reply:
yeah we had a mix of linkedin behavior and leetcode

linked has it reversed. i dont have a pref here so im okay with this but we can iterate if need be

Reply:
Thanks for the background, I'm around if we change our mind :+1: